### PR TITLE
header file added one new tab

### PR DESCRIPTION
**Pull request by truptighatol**

### CODE REVIEW

File: src/app/header/header.component.html [hunk 1]
```diff
@@ -15,6 +15,9 @@ <h1>BookMyShow</h1>
         <li class="nav-item" routerLinkActive="active">
           <a class="nav-link" href="#" routerLink="/people">People</a>
         </li>
+        <li class="nav-item" routerLinkActive="active">
+          <a class="nav-link" href="#" routerLink="/people">Discover</a>
```
Comment (ahghatol) [inline src/app/header/header.component.html]: @truptighatol duplicate router link `/people`, please fix this

Comment (ahghatol) [reply src/app/header/header.component.html]: it should be `/discover` instead `/people`

Comment (truptighatol) [reply src/app/header/header.component.html]: ok

**Comment by truptighatol** [comment]
aalay na

**Comment by truptighatol** [comment]
router changes done

**Pull request merged**